Sources/JWTKeychain/Support/Configuration/Configuration.swift

```diff
@@ -15,67 +15,103 @@ public protocol ConfigurationType {
 
      var signer: String { get }
 
-    static func boot(signer: String, signatureKey: String, publicKey: String?, secondsToExpire: Double) throws
+     func getTokenSignatureKey() -> Bytes
+     func generateExpirationDate() throws -> Date
+     func getSigner(key: Bytes) -> Signer
+     func validateToken(token: String) throws -> Bool
+     func generateToken(userId: Node) throws -> String
 }
 
 public struct Configuration: ConfigurationType {
-    
-    public static var instance: Configuration?;
-    
+
     /// Seconds the JWT has to expire (in the future)
     public var secondsToExpire: Double? = nil
```

Comment:
This needs to be changed to non-nil Double as it is provided in all the initialisers.